app/views/browse/secret.html

```diff
@@ -32,6 +38,8 @@ <h1 class="contains-actions">
               </li>
             </ul>
           </div>
+          <div class="pull-right" ng-hide="!('secrets' | canIDoAny)">
```

Comment:
We should probably just remove the `canIDoAny` check if "Add to Application" is always enabled. Otherwise we need to update the `canIDoAny` filter for secrets.

Reply:
Sorry this was a left over empty div.